dbt/include/tidb/macros/materializations/incremental/helpers.sql

```diff
@@ -1,3 +1,33 @@
+{% macro tidb__get_incremental_default_sql(arg_dict) %}
```

Comment:
this behaviour need to write into the doc

Reply:
@shiyuhang0 Thanks for your review. I will add this behavior to the doc.